MODULE.bazel

```diff
@@ -0,0 +1,24 @@
+module(
+    name = "bazeldnf",
+    version = "0.6.0",
```

Comment:
Is this a value which we need to bump for releases to match the release tag?

Reply:
Maybe something which we can always set to the tag value in the release script with e.g. buildozer?

Reply:
Yes so the version is something that gets bumped with a GitHub release and the Bazel central registry (BCR) provided with GitHub workflows to make releasing of new versions trivial, there's a template rule from which every new rule is forking, I need to look into the details and adopt, once we have those hooks a new release for BCR is just a tag in another public GitHub repo